src/main/java/edu/unc/lib/boxc/migration/cdm/options/PermissionMappingOptions.java

```diff
@@ -0,0 +1,68 @@
+package edu.unc.lib.boxc.migration.cdm.options;
+
+import picocli.CommandLine.Option;
+
+public class PermissionMappingOptions {
+    @Option(names = {"-id", "--cdm-id"},
+            description = {
+                    "CDM ID(s) of the item(s) being mapping"})
+    private String cdmId;
+
+    @Option(names = {"-e", "--everyone"},
+            description = {
+                    "The patron access role assigned to the “everyone” group."})
+    private String everyone;
+
+    @Option(names = {"-a", "--authenticated"},
+            description = {
+                    "The patron access role assigned to the “authenticated” group (anyone that is logged in)."})
+    private String authenticated;
```

Comment:
Optional, but it looks like we might be able to make the types for everyone and authenticated be UserRole:
https://picocli.info/#_enum_types
which might make things easier down in the service, although you'd still have to check if the value is a patron role using `isStaffRole` later.